hca/dss/__init__.py

```diff
@@ -168,17 +172,26 @@ def download_manifest(self, manifest, replica, initial_retries_left=10, min_dela
         """
         Process the given manifest file in TSV (tab-separated values) format and download the files referenced by it.
 
-        Each row in the manifest represents one file in DSS. The manifest must have a header row. The header row must
-        declare the following columns:
+        :param str manifest: path to a TSV (tab-separated values) file listing files to download
+        :param str replica: the replica to download from. [aws,gcp,azure]
+        :param int initial_retries_left: The initial number of times to retries a failed download.
```

Comment:
s/retries/retry

Also, what does "initial" mean here? It should really be "num_retries" or something else more intuitive.

Reply:
added additional details to initial_retries_left